nix/modules/scripts.nix

```diff
@@ -57,7 +49,8 @@
           git add "$VERSIONS_DIR"/flake.lock
         fi
 
-        if [[ "$VERSIONS_DIR" == "$DEFAULT_VERSIONS_DIR" ]]; then
+        # Want to update the root flake.lock if we're updating the version that is currently the default.
+        if grep -q "dir=$VERSIONS_DIR" flake.nix; then
           # TODO, once the Nix version on CI supports it -> nix flake update versions
           nix flake lock --tarball-ttl 0 --update-input versions --override-input versions "path:$VERSIONS_DIR"
```

Comment:
the attribute `.dir` or `.path` (as it's now) in the flaek.lock stems from this line. here we pass `path:...` which will be represented accordingly in the lock file. we just need to match the jq expression and this argument.